### ISSUE TITLE
import and export god roll list

### ISSUE THREAD

**Issue by jsmith0523** [dcaslin/d2-checklist#332]
I'd like to be able to import and export the god roll lists so that I can have customized lists.

D2 Checklist is hands down the best UI if you want to review weapons for god rolls based on specific perks. However the more I do it, I'm realizing some of my preferences deviate from Panda's. As is I actually have to review the perks and mentally overlay my preferences on what the UI is telling me. If I could edit my preferences into the system it would make reviewing rolls a lot faster.

**Comment by jsmith0523** [comment]
Hopefully I didn't overstep by directly submitting the issue. I wouldn't mind implement it myself or contributing in general but life just won't allow it right now.

**Comment by dcaslin** [comment]
All good! It's a good request and one I've gotten elsewhere. I actually used to support it but dropped support when I moved to a more advanced god roll format that differentiated between MnK and keyboard (among other improvements). It may be a while before i get to it, but I will try to get to it. Easiest implementation path would be downloading a pretty print of the JSON doc (i.e. this one) https://github.com/dcaslin/d2-checklist/blob/master/src/assets/panda-godrolls.min.json, then letting you re-upload it and using that until you told it to stop. Would that be helpful for you?

**Comment by jsmith0523** [comment]
Yes, it looks pretty easy to decipher. Even providing it as-is might work, I believe Notepad++ can sort out the pretty print itself.

**Comment by jsmith0523** [comment]
Looks like this should be OBE with PerkBench the way it is. Only thing that would make it a lot more usable is some filtering so I don't have to load every single weapon possible on a screen.
